### ISSUE TITLE
[CHAINWHIZ]- Latest bounty segment

### ISSUE THREAD

**Issue by akp111** [ChainWhiZ/chainwhiz-latest-bounty#1]
Title : Implement the figma design to showcase the latest bounties

Figma Link: https://www.figma.com/file/VCR5loBhRE2m4w4sizOTgX/Latest-Bounties?node-id=0%3A1

Description : Use this react app as template to implement the latest bounty page. It should be dynamic enough such that latest 3 active bounties will be visible. Use Material UI whenever needed. The page should be responsive in desktop screens (preferable screen resolutions: 2560 × 1600, 1920 x 1080). Use Javascript to code.

API Link: https://backend-app.chainwhiz.app/question/fetchall

Evaluation Criteria:

Use vh, vw for widths and heights, and rem for attributes like font size, margins, etc. Use px only where needed.
Must be responsive atleast in the given resolutions
Max use of Material UI
Good to have mobile responsive but not mandatory
No hardcoded data.

**Comment by codeWhizperer** [comment]
Hello @akp111 I am interested in working on this project. But i have a few questions

**Comment by xlassix** [comment]
Hello @akp111, I also have some questions, after going through the requirements I would love to take on the bounty a condition has to be met for easy integration with the current system. 
    A scaffold project with colours schemes or theme config or project config of what is currently deployed or the bounty will be added to, this is to avoid spending time creating a well-optimized or top-tier standard contribution and getting it tossed because of compatibility issues. 

Thank you.

**Comment by ASdevix** [comment]
Hey, @Adegbite1999 , please ask all your queries here. We're happy to answer.

**Comment by codeWhizperer** [comment]
I have gone through the figma design and it's quite straight forward. Does
the bounty also include integrating the Api endpoints. Because checking
through the scheme there's more information in the endpoints as compared to
the figma design. I just want to clarify before I proceed with the task.

**Comment by ASdevix** [comment]
Yes, @Adegbite1999 you will have to use the API endpoints but you have to filter out the questions which have `questionStage` as `completed` as these won't be displayed. Good luck with the task!

**Comment by ASdevix** [comment]
> 

Hey, @xlassix - We use MaterialUI components and for styling, we use CSS classes. As for themes and colors, you can find them using the Figma link. Good luck with the task!

**Comment by xlassix** [comment]
Hey @ASdevix, by configurations I meant more key details like is the project is using typeScript or not or perhaps some configuration files like ```package.json```, ```ts.config```, ```.eslintrc.js``` or theme configuration(to avoid hard coding color codes). Creating a pixel-Perfect implementation using MaterialUI isn't an issue, as they say, the devil is in the details

**Comment by mishramonalisha76** [comment]
Hi @xlassix I have added the template we are using. You can refer that.

**Comment by xlassix** [comment]
Thanks @mishramonalisha76, I would work with that. even though the files are obviously from a fresh project init 🤣

**Comment by ASdevix** [comment]
Good luck @xlassix
